utils/netwhere/Makefile

```diff
@@ -0,0 +1,39 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=netwhere
+PKG_VERSION:=0.5
+PKG_RELEASE:=1
+PKG_LICENSE:=MIT
+
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
```

Comment:
This is obviously a bad idea and will clash with things in the end. Since you're the author create a proper tarball (preferably tar.xz) and link to that instead or pull the code (please avoid this if you can).